assignment_template/Analysis.py

```diff
@@ -27,11 +28,45 @@ def load_data(self) -> None:
         print(self.config['figure_title'])
 
     def compute_analysis(self) -> Any:
-        pass
+        '''Analyze previously-loaded data.
+
+        This function compute the mean of pokemons' base experience 
+
+        Parameters
+        ----------
+        None
+
+        Returns
+        -------
+        analysis_output : Float
+
+        '''
+        return np.mean(self.dataset["base_experience"])
 
     def plot_data(self, save_path: Optional[str] = None) -> plt.Figure:
         pass
 
     def notify_done(self, message: str) -> None:
-        pass
+        ''' Notify the user that analysis is complete.
+
+        Send a notification to the user through the ntfy.sh webpush service.
+
+        Parameters
+        ----------
+        message : str
+        Text of the notification to send
+
+        Returns
+        -------
+        None
+
+        '''
+        topic = 'building-software-assignemt-analysis'
```

Comment:
This will have to come from the config. Probably this pull request will have to be modified after the config part is done.  I will add the config with the init work soon.

Reply:
I'll add ntfy topic name to user config soon.